core/templates/dev/head/services/image-upload-helper.service.ts

```diff
@@ -52,7 +52,7 @@ angular.module('oppia').factory('ImageUploadHelperService', [
             entityType, entityId, encodedFilepath));
       },
 
-      generateImageFilename: function() {
+      generateImageFilename: function(height, width) {
```

Comment:
Probably cleaner to pass fileExtension in as well, since that's part of the filename.

There should probably be unit tests written for this function too?

Reply:
Done.